spire/templates/shared-alb.yml

```diff
@@ -435,30 +435,44 @@ Resources:
           SetIdentifier: !Ref AWS::StackName
           Type: A
 
-  # Waf:
-  #   Type: AWS::WAFv2::WebACL
-  #   Properties:
-  #     DefaultAction:
-  #       Allow: {}
-  #     Description: !Sub WAF for Spire ${EnvironmentType} shared ALB
-  #     Scope: REGIONAL
-  #     Tags:
-  #       - { Key: prx:meta:tagging-version, Value: "2021-04-07" }
-  #       - { Key: prx:cloudformation:stack-name, Value: !Ref AWS::StackName }
-  #       - { Key: prx:cloudformation:stack-id, Value: !Ref AWS::StackId }
-  #       - { Key: prx:cloudformation:root-stack-name, Value: !Ref RootStackName }
-  #       - { Key: prx:cloudformation:root-stack-id, Value: !Ref RootStackId }
-  #       - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
-  #       - { Key: prx:dev:application, Value: Common }
-  #     VisibilityConfig:
-  #       CloudWatchMetricsEnabled: false
-  #       MetricName: !Sub ${Alb.LoadBalancerName}-WAF
-  #       SampledRequestsEnabled: false
-  # WafAssociation:
-  #   Type: AWS::WAFv2::WebACLAssociation
-  #   Properties:
-  #     ResourceArn: !Ref Alb
-  #     WebACLArn: !GetAtt Waf.Arn
+  Waf:
```

Comment:
Do we want to put this in its own template, so we can more easily share it between this ALB and the DT ALB?